### DISCUSSION TITLE
Ingestion Log doesn't scroll

### DISCUSSION THREAD

**Discussion by mack0196** [datalust/seq-tickets#1406] [Help]
When the Ingestion Log Diagnostic page is open it says 'This log shows the last 100 events from Seq's ingestion endpoints,'

I have 2 and a partial 3rd log on the screen but it doesn't scroll so I cannot see much.

**Comment by nblumhardt** [comment]
Thanks for the heads-up, it appears we've broken the layout of this page, I've raised https://github.com/datalust/seq-tickets/issues/1407.

You can work around this and view the whole log by replacing `#` and everything after it in your browser's address bar with `/api/diagnostics/ingestion`, e.g. `https://seq.example.com/api/diagnostics/ingestion`.

Let me know if this helps!
